README.md

```diff
@@ -32,6 +32,12 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
```

Comment:
If you are using `SECRET_KEY` for your Trello credentials, you should remove the mention of it here. And you should remove it from flask_config.py

It was there previously for encrypting session cookies.